README.md

```diff
@@ -7,12 +7,40 @@
  - You can add a book.
  - It will display your added books.
  - It has a button to remove a book.
+ - Books are saved in localStorage.
+ - It has a style following the given general layout.
+=======
  <!-- - It has a button to change the read status of the book. -->
 
 ## Built With
 
 - Html, CSS, and JavaScript
 
+## Live Demo
+
+[Site](https://anapdh.github.io/awesome-books/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps:
+
+### Install
+
+You can download the code to your personal computer cloning the repository using the following command:
+
+```
+git clone
+```
+
+### Usage
+
+To run this project you only need to open the index.html file in a web browser.
+
+## Authors
+
+👩🏼‍💻 **Ana Paula Hübner**
+
+=======
```

Comment:
You could consider removing this part on your README file because there is a repetition below it. This helps us to achieve a neat and professional repo.